api/rhsm-subscriptions-api-spec.yaml

```diff
@@ -654,6 +654,33 @@ components:
           schema:
             $ref: "#/components/schemas/Errors"
   schemas:
+    GranularityGenerated:
```

Comment:
Opinions on name for this?  I tacked on "Generated" at the end to circumvent the competing namespace with the db.model.Granularity class.  Partly because of checkstyle and lines being too long when you fully qualify the package, as well as just general confusion having two things with the same name.  The side effect is that it shows up like this in the api-doc page now.  Thoughts? Opinions?

![image](https://user-images.githubusercontent.com/19955562/101080015-3ebe3d00-3576-11eb-9f62-7f8fee2bf462.png)

Reply:
I like `GranularityType`, `ServiceLevelType`, etc.  `Generated`.  I think putting a comment above the schema definition about wanting to avoid naming conflicts with the DB model classes is worth having also.  Otherwise, someone might be apt to shorten it in a refactor.

Reply:
I like it!